### PR TITLE
[FAB-17263] Update upgrade doc with peer docker command

### PR DESCRIPTION
**Pull request by suryalnvs**
Signed-off-by: Surya Lanka <suryalnvs@gmail.com>

#### Type of change

- Documentation update

#### Description

Update peer docker commands in upgrade docs to:
```
docker run -d -v /opt/backup/$PEER_CONTAINER/:/var/hyperledger/production/ \
            -v /opt/msp/:/etc/hyperledger/fabric/msp/ \
            --env-file ./env<name of node>.list \
            --name $PEER_CONTAINER \
            hyperledger/fabric-peer peer node start
```
in https://github.com/hyperledger/fabric/blob/master/docs/source/upgrade_to_newest_version.md and https://github.com/hyperledger/fabric/blob/master/docs/source/upgrading_your_components.md

#### Related issues

https://jira.hyperledger.org/browse/FAB-17263

### CODE REVIEW

**Review by joealewine — DISMISSED** [review]

File: docs/source/upgrading_your_components.md [hunk 1]
```diff
@@ -29,8 +29,13 @@ This tutorial assumes a Docker deployment where the YAML files will be baked int
...
 When you deploy a peer or an ordering node, you had to set a number of environment variables relevant to its configuration. A best practice is to create a file for these environment variables, give it a name relevant to the node being deployed, and save it somewhere on your local file system. That way you can be sure that when upgrading the peer or ordering node you are using the same variables you set when creating it by issuing:
 
+For peer:
 ```
-docker run --env-file ./env<name of node>.list ubuntu bash
+docker run -d -v /opt/backup/$PEER_CONTAINER/:/var/hyperledger/production/ \
+            -v /opt/msp/:/etc/hyperledger/fabric/msp/ \
+            --env-file ./env<name of node>.list \
+            --name $PEER_CONTAINER \
+            hyperledger/fabric-peer peer node start
```
Comment (sykesm) [inline docs/source/upgrading_your_components.md:38]: Talked to Joe about the dismissed review comment. He was asking about the image versions in the command. It appears we use it in other places but not here.

Should we have a version?

File: docs/source/upgrading_your_components.md [hunk 2]
```diff
@@ -53,6 +58,15 @@ CORE_PEER_GOSSIP_EXTERNALENDPOINT=peer0.org1.example.com:7051
 CORE_PEER_LOCALMSPID=Org1MSP
 ```
 
+For ordering node:
+```
+docker run -d -v /opt/backup/$ORDERER_CONTAINER/:/var/hyperledger/production/orderer/ \
+            -v /opt/msp/:/etc/hyperledger/fabric/msp/ \
+            --env-file ./env<name of node>.list \
+            --name $ORDERER_CONTAINER \
+            hyperledger/fabric-orderer orderer
```
Comment (sykesm) [inline docs/source/upgrading_your_components.md:67]: Should there be an image version here?

File: docs/source/upgrading_your_components.md (hunk 1 above)
Comment (joealewine) [reply docs/source/upgrading_your_components.md:38]: Talked to Surya. I'm fine with getting these merged for now, then I'll go back and add a bit more text and environment variables for these sections where needed in a subsequent PR.

File: docs/source/upgrading_your_components.md (hunk 2 above)
Comment (joealewine) [reply docs/source/upgrading_your_components.md:67]: See above.

**Review by sykesm — APPROVED** [review]

**Pull request merged**